aspnetcore/client-side/bundling-and-minification.md

```diff
@@ -86,36 +61,120 @@ imageTagAndImageID | t
+* minify - minification options for the output type. **optional**, *default - `minify: { enabled: true }`*
+  * Configuration options are available per output file type.
+    * [CSS Minifier](https://github.com/madskristensen/BundlerMinifier/wiki/cssminifier)
+    * [JavaScript Minifier](https://github.com/madskristensen/BundlerMinifier/wiki/jsminifier)
+    * [HTML Minifier](https://github.com/madskristensen/BundlerMinifier/wiki/htmlminifier)
+* includeInProject - add generated files to project file. **optional**, *default - false*
+* sourceMaps - generate source maps for the bundled file. **optional**, *default - false*
 
-## Impact of bundling and minification
+### Visual Studio 2015 / 2017
 
-The following table shows several important differences between listing all the assets individually and using bundling and minification on a simple web page:
+Open *bundleconfig.json* in Visual Studio, if your environment does not have the extension installed; a prompt is presented suggesting that there is one that could assist with this file type.
 
-Action | With B/M | Without B/M | Change
---- | :---: | :---: | :---:
-File Requests |7 | 18 | 157%
-KB Transferred | 156 | 264.68 | 70%
-Load Time (MS) | 885 | 2360 | 167%
+![BuildBundlerMinifier Extension Suggestion](../client-side/bundling-and-minification/_static/bundler-extension-suggestion.png)
 
-The bytes sent had a significant reduction with bundling as browsers are fairly verbose with the HTTP headers that they apply on requests. The load time shows a big improvement, however this example was run locally. You will get greater gains in performance when using bundling and minification with assets transferred over a network.
+Select View Extensions, and install the **Bundler & Minifier** extension (Requires Visual Studio restart).
+
+![BuildBundlerMinifier Extension Suggestion](../client-side/bundling-and-minification/_static/view-extension.png)
+
+When the restart is complete you need to configure build to run the processes of minifying and bundling the client side assets. Right click the *bundleconfig.json* file and select *Enable bundle on build...*.
+
+Build the project and the *bundleconfig.json* is included in the build process to produce the output files based on the configuration.
+
+```console
+1>------ Build started: Project: BuildBundlerMinifierExample, Configuration: Debug Any CPU ------
+1>
+1>Bundler: Begin processing bundleconfig.json
+1>Bundler: Done processing bundleconfig.json
+1>BuildBundlerMinifierExample -> C:\BuildBundlerMinifierExample\bin\Debug\netcoreapp1.1\BuildBundlerMinifierExample.dll
+========== Build: 1 succeeded or up-to-date, 0 failed, 0 skipped ==========
+```
+
+### Visual Studio Code or Command Line
+
+Visual Studio and the extension drives the process and using GUI gestures, however the same capabilities are available with the `dotnet` CLI and BuildBundlerMinifier NuGet Package.
```

Comment:
> Visual Studio and the extension drives the process and using GUI gestures

Visual Studio and the extension drive the bundling and minification process using GUI gestures